example/randomCells2D/src/randomCells2D.cpp

```diff
@@ -224,8 +218,9 @@ auto main() -> int
         }
     }
 
+    BufAcc bufAccS{alpaka::zero_memcpy(queue, bufHostS)};
+    float* const ptrBufAccS{alpaka::getPtrNative(bufAccS)};
     auto pitchBufAccS = alpaka::getPitchBytes<1u>(bufAccS);
-    alpaka::memcpy(queue, bufAccS, bufHostS);
```

Comment:
Buffer creation is moved down to where the memory copies were performed, because allocation/memcpy is now one potential step hidden behind `zero_memcpy`.